skxray/speckle_visibility/speckle_visibility.py

```diff
@@ -0,0 +1,290 @@
+# ######################################################################
```

Comment:
@tacaswell Ok sure, I renamed and moved the file to top level, ready for PR review